.circleci/config.yml

```diff
@@ -48,7 +48,7 @@ commands:
       - run:
           name: Rust Clippy
           command: |
-            cargo clippy --all --all-targets --all-features -- -D warnings
+            cargo clippy --workspace --all-targets --all-features -- -D warnings
```

Comment:
[`--all` is a deprecated alias for `--workspace'](https://doc.rust-lang.org/cargo/commands/cargo-test.html#package-selection)